documentation/modules/retrieving-validation-service-json.adoc

```diff
@@ -19,6 +19,20 @@ You can create a validation rule based on any attribute in the `"input"` key, fo
 $ {oc} get route <inventory_service> -n {namespace}
 ----
 
+. Retrieve the access token:
++
+[source,terminal]
+----
+$ TOKEN = $(oc whoami -t)
```

Comment:
There shouldn't be space before and after the "="

Reply:
OK to all of the above.